packages/teams-js/prepBetaRelease.js

```diff
@@ -167,13 +176,17 @@ function getNewPkgJsonContent(currNextDevVer) {
   return packageJson;
 }
 
-function prepNewDevRelease(devStdout) {
+function prepBetaRelease(devStdout) {
   const newPackageJson = getNewPkgJsonContent(devStdout);
   const newVersion = newPackageJson.version;
   saveJsonFile(newPackageJson);
   return newVersion;
 }
 
 (() => {
-  exec(`npm view @microsoft/teams-js version --tag next-dev`).then(({ stdout, stderr }) => prepNewDevRelease(stdout.trim()));
+  exec(`yarn beachball bump`).then(
+    exec(`npm view @microsoft/teams-js version --tag beta`).then(({ stdout, stderr }) =>
```

Comment:
Okay, this is a little tricky. We will have to find a way to manually set the first 2.0 beta version. Since `npm view @microsoft-teams-js version --tag beta` currently returns "1.12.0-beta.5". The first time we do this, I think it needs to query for "--tag next-dev", which would return "2.0.0-beta.7-dev.14".

Reply:
So yarn beachball bump will return 2.1.0, and so I think this will be fine since it will automatically set it to 2.1.0-beta.0